lib/sequelize.js

```diff
@@ -228,6 +231,12 @@ class Sequelize {
 
     this.dialect = new Dialect(this);
     this.dialect.QueryGenerator.typeValidation = options.typeValidation;
+    if (this.options.operatorsAliases) {
```

Comment:
As I said above this should be renamed to just `operators`

Reply:
1. If operators is set to `true`, which will be default value, load all legacy operators
2. If operators is set to an map, set only those operators
3. if operators is set to false or empty value, disable legacy op completely

(1) Will give deprecation warning its also the default state when we release this feature
(2) and (3) will give no warning

Warning should say `String based operators are now deprecated. Please use Symbol based operators for better security, read more at http://docs.sequelizejs.com/manual/tutorial/querying.html#operators`